src/visualization/plot_recipes.jl

```diff
@@ -258,6 +308,26 @@ end
 end
 
 
+@recipe function f(pm::PlotMesh2D{<:PlotData2D{<:Any, <:AbstractVector{<:AbstractVector}}})
+  @unpack plot_data = pm
+  @unpack x, y, mesh_vertices_x, mesh_vertices_y = plot_data
+
+  # Set geometric and annotation properties
+  xlims --> (minimum(x), maximum(x))
+  ylims --> (minimum(y), maximum(y))
+  aspect_ratio --> :equal
+  legend -->  :none
+
+  # Set series properties
+  seriestype := :path
+  linecolor := :black
+  linewidth := 1
```

Comment:
```suggestion
  seriestype --> :path
  linecolor --> :black
  linewidth --> 1
```
To allow user overrides